dali/operators/reader/loader/file_loader.h

```diff
@@ -43,7 +43,7 @@ template <typename Backend = CPUBackend, typename Target = FileWrapper,
           typename InputStream = FileStream>
 class FileLoader : public Loader<Backend, Target> {
  public:
-  explicit inline FileLoader(const OpSpec &spec, bool shuffle_after_epoch = false)
+  explicit inline FileLoader(const OpSpec &spec, bool shuffle_after_epoch)
```

Comment:
Nitpick: `explicit` may be removed here. Also, `inline` since this function is by default inline:

> A function defined entirely inside a [class/struct/union definition](https://en.cppreference.com/w/cpp/language/classes), whether it's a member function or a non-member friend function, is implicitly an inline function unless it is attached to a [named module](https://en.cppreference.com/w/cpp/language/modules#Module_declarations) (since C++20).

https://en.cppreference.com/w/cpp/language/inline

Reply:
Done